app/views/versions/_close_ui.html.erb

```diff
@@ -12,7 +12,12 @@
     <label for="description">Version description</label><br>
     <textarea id="description" name="description" class='form-control'><%= @description %></textarea>
   </div>
-  <button type='submit' class='btn btn-primary'>
-    Close Version
-  </button>
+  <% if @cocina_object.instance_of?(NilModel) %>
+    <div class='alert alert-danger'>This object cannot currently be represented in the Cocina model and thus should
```

Comment:
I'm wondering if it makes sense to show any of the form. I think that is a little bit confusing that  it lets you can enter values, but you can't save them.  Also consider `alert-warn` rather than `alert-danger`.